doc/dev/crimson/crimson.rst

```diff
@@ -176,6 +176,68 @@ see `Prometheus`_ for more details.
 Profiling Crimson
 =================
 
+fio
+---
+
+``crimson-store-nbd`` exposes SeaStore as an NBD server for use with fio in
+basic benchmarking.
```

Comment:
It actually can expose alienstore with --type = "bluestore".  See fs_driver.cc, config.get_fs_type().  How about "exposes configurable FuturizedStore internals as an NBD server for use with fio."

Reply:
updated.